cg/store/crud/create.py

```diff
@@ -541,6 +541,9 @@ def create_pac_bio_sample_sequencing_run(
         sample_id: str = sample_run_metrics_dto.sample_internal_id
         LOG.debug(f"Creating Pacbio sample sequencing metric for sample {sample_id}")
         sample: Sample = self.get_sample_by_internal_id(sample_id)
+        if not sample:
+            self.session.rollback()
+            raise EntryNotFoundError(f"Sample not found: {sample_id}")
```

Comment:
I get that we have not seen the error thus far but would it make sense to do the same thing with the sequencing run?

Reply:
the sequencing run is an aggregate of the samples and is called donwstream of this. It will fail here first and not proceed further. 

I will leave this as is for now as the plan is to move the actual fail into the crud layer instead of layers using the crud. So then it will be checked and required in one place only. The current situation is not DRY and coupled.